detox/src/client/AsyncWebSocket.js

```diff
@@ -12,6 +12,7 @@ class AsyncWebSocket {
     this.inFlightPromises = {};
     this.eventCallbacks = {};
     this.messageIdCounter = 0;
+    this.timeout = 60000;
```

Comment:
Pls use the Detox configurable timeout, not magic numbers.

Reply:
Okay, I hope I'll find a way to do it.

Reply:
We discussed a way to use the test runner timeout parameter, it can be added to the test runner adapter for easier extraction